addons/website/tests/test_create_website.py

```diff
@@ -0,0 +1,33 @@
+from unittest.mock import patch
+from odoo.tests import HttpCase
+import odoo.tests
+
+@odoo.tests.common.tagged('post_install', '-at_install')
+class TestCreateWebsite(HttpCase):
+
+    def _theme_upgrade_upstream(self):
+        # Actions are quite tricky in this flow.
+        # Because we cannot do _theme_upgrade_upstream
+        # the theme install action isn't consummed
+        # and it puts the client back to choosing a theme
+        # once it just picked one. So we make sure only the
+        # usual action is enabled
+        actions = self.env['ir.actions.todo'].search([('state', '=', 'open')])
+        actions.write({'state': 'done'})
+        self.env['ir.actions.todo'].create({'action_id': self.env.ref('website.action_website_edit').id, 'state': 'open'})
+
+    def setUp(self):
+        super().setUp()
+        patcher = patch('odoo.addons.website.models.ir_module_module.IrModuleModule._theme_upgrade_upstream', wraps=self._theme_upgrade_upstream)
+        patcher.start()
+        self.addCleanup(patcher.stop)
+
+    def test_01_create_website(self):
+        self.env['res.lang'].create({
+            'name': 'Parseltongue',
+            'code': 'pa_GB',
+            'iso_code': 'pa_GB',
+            'url_code': 'pa_GB',
+        })
+        self.start_tour('/', 'test_create_website_1', login='admin')
+        self.start_tour('/web#action=website.action_website_configuration', 'test_create_website_2', login='admin')
```

Comment:
Any reason not to do that in a single test? (I mean, you could concat the steps and it would be only a single test right?)
Starting a tour is expensive, it spawns a new browser, go through auth again, has to load the routing map/cache etc

Reply:
Also, this might be very useful to add the info about the reason why it was failing, that it is actually because of the previous website visit that generated a lang cookie.
If you can and concat the tours so it becomes only one, I'd add a line between the steps like
```
// Now go through the configurator for a new website. Previous steps created a frontend lang cookie which should not be used when redirecting to the frontend after the configurator in the following steps.
```
(Something like that, you got the idea, but I think it helps making the tour more understandable if you are not the person involved in its creation, in case you have to work on it later)

Reply:
Is there a way to change URL within a tour ? Issue I have is that I'd need to go to settings and that was different between Community and Enterprise.

Reply:
Just doing a regular JS redirect with the same url as in your python tour on this line
```js
content: "Go in backend settings",
trigger: 'body',
run: function () {
    window.location.href = '/web#action=website.action_website_configuration';
},
```